### PR TITLE
Add search button

### PR DESCRIPTION
**Pull request by palakjadwani**
**Fixes issue:**
[#31]

**Changes:**
- Added search button
- Fixed footer issue
- Changed header appearance 

screenshot 333

### CODE REVIEW

File: code/popup.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
   <link rel="stylesheet" type="text/css" href="css/font.css">
   <link rel="stylesheet" type="text/css" href="css/fontawesome-all.min.css">
   <link rel="shortcut icon" type="image/png" href="/icon/icon.png"/>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (AdiChat) [inline code/popup.html]: We can link only to local resources as this extension is intended to work offline. We are using `fontawesome-all.min.css`, so, you may consider using this as we must consider the size of our extension as well.

**Comment by AdiChat** [comment]
The UI looks good. :+1: 

I have added a comment regarding a small change. Kindly review it and make corresponding changes.

**Comment by palakjadwani** [comment]
Yes okay. ^_^

**Comment by palakjadwani** [comment]
Some things got messed up hence created a separate PR. So I am closing this one. :)
Refer to https://github.com/OpenGenus/quark/pull/89/

**Pull request closed without merging**